projects/plugins/protect/src/class-jetpack-protect.php

```diff
@@ -252,6 +252,18 @@ public function admin_bar( $wp_admin_bar ) {
 	 * @return void
 	 */
 	public static function register_rest_endpoints() {
+		register_rest_route(
+			'jetpack-protect/v1',
```

Comment:
why do we register a new endpoint when we could use `/my-jetpack/v1/sites/product/$product` instead?